packages/expo/src/winter/fetch/convertFormData.ts

```diff
@@ -30,6 +30,20 @@ export async function convertFormDataAsync(
       results.push(entry.file.bytes());
     } else if ('blob' in entry) {
       results.push(new Uint8Array(await blobToArrayBufferAsync(entry.blob)));
+    } else if (entry._data?.blobId != null) {
+      // When `FormData.getParts()` is called, React Native will use spread operator to copy the object and loose the Blob type info.
```

Comment:
(spread is not an operator, technically)
```suggestion
      // When `FormData.getParts()` is called, React Native will use the spread syntax to copy the object and lose the Blob type info.
```